### PR TITLE
COMMERCE-9387 Dynamic field fragment

### PR DESCRIPTION
**Pull request by lorenzolapescara**
https://issues.liferay.com/browse/COMMERCE-9387
https://issues.liferay.com/browse/COMMERCE-9386
https://issues.liferay.com/browse/COMMERCE-9388
https://issues.liferay.com/browse/COMMERCE-9389

Re-sent of https://github.com/liferay-commerce/liferay-portal/pull/3496

### CODE REVIEW

**Comment by lorenzolapescara** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceDynamicFieldFragment.testcase [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+				fsOptionName = "Package Quantity",
+				fsOptionValue = 48);
+		}
+
+		task ("Then the Dynamic Field fragments update and values are shown based on the selected option") {
+			CommerceDisplayPageTemplates.viewDynamicFields(valuesList = "Inventory:240,SKU:MIN93022C,GTIN:GTIN0003,Manufacturer Part Number:MPN0003");
+		}
+	}
+
+	@description = "COMMERCE-11075 | Using Dynamic Field fragment, I want to be able to change the HTML element where the label and the value are shown"
```
Comment (baharturk) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceDynamicFieldFragment.testcase:140]: Please use the same as you did for the other test, using ".", also in the end.

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceDynamicFieldFragment.testcase [hunk 2]
```diff
@@ -0,0 +1,224 @@
+@component-name = "portal-commerce"
+definition {
+
+	property custom.properties = "jsonws.web.service.paths.excludes=${line.separator}company.security.strangers.verify=false${line.separator}feature.flag.COMMERCE-5898=true";
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property release.feature.flags.disable.DISABLE_PRIVATE_LAYOUTS = "true";
```
Comment (baharturk) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceDynamicFieldFragment.testcase:7]: It's not needed for this file.

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceDynamicFieldFragment.testcase [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+
+		task ("Given GTIN and MPN fields are added to a sku") {
+			CommerceJSONProductsAPI._patchCommerceSku(
+				discontinued = "false",
+				gtin = "GTIN0001",
+				manufacturerPartNumber = "MPN0001",
+				productSku = "MIN55861");
+		}
+
+		task ("And 4 Dynamic Field fragments are deployed withdifferent Label Element Types and Value Element Types for each fragment") {
```
Comment (baharturk) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceDynamicFieldFragment.testcase:155]: Please add a space between "withdifferent".

**Review by baharturk — CHANGES_REQUESTED** [review]

**Comment by lorenzolapescara** [comment]
ci:test:commerce-functional

**Comment by lorenzolapescara** [comment]
ci:test:relevant

**Comment by alessiorendina** [comment]
ci:forward

**Pull request closed without merging**